openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -142,7 +142,7 @@
   }
 
   // actions
-  .action--submit, .action--save {
+  .action--submit, .action--save, .action--upload {
     text-shadow: none !important;
```

Comment:
FYI, here you can see the constellation of different buttons:

![image](https://cloud.githubusercontent.com/assets/484484/20154768/efee092a-a696-11e6-8695-49c08081decb.png)